### ISSUE TITLE
Unable to Get Simple Script to Work

### ISSUE THREAD

**Issue by abstractspoon** [unvell/ReoGrid#465]
Hi Unvell

I've spent a couple of evenings trying to make progress with scripting but without luck.

I'm running the Winforms Editor Project with EX_SCRIPT defined.
![image](https://user-images.githubusercontent.com/12944354/168416470-e5561c21-bf08-4b30-b459-161142e4c5b5.png)

 and have entered the following script (from the [documentation](https://reogrid.net/document/chart/line-chart/)) into the Script Editor:
```
// get current worksheet
var worksheet = workbook.currentWorksheet;

var dataRange = new worksheet.Ranges["B3:F5"];
var serialNamesRange = worksheet.Ranges["A3:A6"];
var categoryNamesRange = worksheet.Ranges["B2:F2"];

worksheet.AddHighlightRange(categoryNamesRange);
worksheet.AddHighlightRange(serialNamesRange);
worksheet.AddHighlightRange(dataRange);
```
but when I attempt to 'run' the script I just get: 
```
error: Attempt to access undefined array or object
```
![image](https://user-images.githubusercontent.com/12944354/168416662-9769ec84-4c5a-4675-b5f8-c9ffeb46f147.png)

Is the documentation out of date or am I doing something wrong?

**Comment by jingwood** [comment]
You can't run C# code from the Script Editor, which can only run ReoScript code.
See https://reogrid.net/document/script-execution/

**Comment by abstractspoon** [comment]
Thanks for that.

Could you possibly fix the Tetris.reo example (code is truncated) or are there any other example scripts which might be useful?

**Comment by qq8344310** [comment]
T



---Original---
From: ***@***.***&gt;
Date: Thu, Jan 12, 2023 12:59 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [unvell/ReoGrid] Unable to Get Simple Script to Work (Issue #465)




 
Closed #465 as completed.
